exercises/practice/triangle/triangle_test.rb

```diff
@@ -86,28 +86,40 @@ def test_isosceles_triangle_sides_may_be_floats
     assert triangle.isosceles?, "Expected 'true', triangle [0.5, 0.4, 0.5] is isosceles."
   end
 
-  def test_triangle_is_scalene_if_no_sides_are_equal
+  def test_scalene_triangle_no_sides_are_equal
     skip
     triangle = Triangle.new([5, 4, 6])
     assert triangle.scalene?, "Expected 'true', triangle [5, 4, 6] is scalene."
   end
 
-  def test_triangle_is_not_scalene_if_all_sides_are_equal
+  def test_scalene_triangle_all_sides_are_equal
     skip
     triangle = Triangle.new([4, 4, 4])
-    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 4] is not scalene."
+    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 4] is scalene."
   end
 
-  def test_triangle_is_not_scalene_if_two_sides_are_equal
+  def test_scalene_triangle_first_and_second_sides_are_equal
     skip
     triangle = Triangle.new([4, 4, 3])
-    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 3] is not scalene."
+    refute triangle.scalene?, "Expected 'false', triangle [4, 4, 3] is scalene."
   end
 
-  def test_sides_that_violate_triangle_inequality_are_not_scalene_even_if_they_are_all_different
+  def test_scalene_triangle_first_and_third_sides_are_equal
+    skip
+    triangle = Triangle.new([3, 4, 3])
+    refute triangle.scalene?, "Expected 'false', triangle [3, 4, 3] is scalene."
+  end
+
+  def test_scalene_triangle_second_and_third_sides_are_equal
+    skip
+    triangle = Triangle.new([4, 3, 3])
+    refute triangle.scalene?, "Expected 'false', triangle [4, 3, 3] is scalene."
+  end
+
+  def test_scalene_triangle_may_not_violate_triangle_inequality
     skip
     triangle = Triangle.new([7, 3, 2])
-    refute triangle.scalene?, "Expected 'false', triangle [7, 3, 2] is not scalene."
+    refute triangle.scalene?, "Expected 'false', triangle [7, 3, 2] is scalene."
```

Comment:
If we are refuting that a triangle is scalene, with then should it not be scalene?